nuget/nunitlite.nuspec

```diff
@@ -11,8 +11,19 @@
     <iconUrl>http://nunit.org/nuget/nunitv3_32x32.png</iconUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <summary>NUnitlite is a lightweight runner for tests that use the NUnit testing framework.</summary>
-    <description>NUnitLite provides a simple way to run NUnit tests, without the overhead of a full NUnit installation. It is suitable for projects that want to have a quick way to run tests using a console runner and don't need all the features of the NUnit engine and console runner. This package contains builds of the NUnitLite runner for .NET 2.0, 4.0 and 4.5 as well as Silverlight 5.0.&#10;&#13;How to use this package:&#10;&#13;1. Create a console application for your tests and delete the generated class containing Main().&#10;&#13;2. Install the NUnitLite package, which creates a new Main() as well as adding a reference to NUnitLite. This will also bring in the NUnit package, which adds a reference to the nunit framework to your project.&#10;&#13;3. Add your tests to the test project and simply start the project to execute them.</description>
-    <releaseNotes>This is an beta release, but is ready for production use for people with prior NUnit experience.</releaseNotes>
+    <description>NUnitLite provides a simple way to run NUnit tests, without the overhead of a full NUnit installation. It is suitable for projects that want to have a quick way to run tests using a console runner and don't need all the features of the NUnit engine and console runner.
+
```

Comment:
It is much more readable with real linefeeds. Saw it done this way while looking at other projects and tested in Visual Studio.